File: HelloWorld.py

print "HelloWorld"

def plus(a,b):
    return a+b

a=10
b=20
c=plus(a,b)
print "{0}{1}".format("plus=",c)

class Sample:
    def minus(self,a,b):
        return a-b

sample = Sample()
d=sample.minus(a,b)
print "{0}{1}".format("minus=",d)

File: HelloWorld.js
"use strict"

console.log("HelloWorld");
var a=10,b=20;
var c=plus(a,b);
console.log("plus=%d",c);

function plus(a,b) {
    return a+b;
}

class HelloWorld {
    minus(a,b) {
        return a-b;
    }
}

let helloworld = new HelloWorld();
var d=helloworld.minus(a,b);
console.log("minus=%d",d);
